BootloaderCommonPkg/BootloaderCommonPkg.dec

```diff
@@ -197,6 +229,14 @@
   # @Prompt Minimum length(in bytes) of the system preboot TCG event log area(LAML).
   gPlatformCommonLibTokenSpaceGuid.PcdTcgLogAreaMinLen|0x10000|UINT32|0x00010081
 
+[PcdsDynamic]
```

Comment:
What's the reason to use Dynamic?

Reply:
Hash/PCR to be extended in Measured boot is provided as option configdata blob.  PcdMeasuredBootHashType would be updated by reading config data TPM alg.
Current added support in TGL platform and SblPlatform merge request is https://gitlab.devtools.intel.com/OWR/IoTG/BDP/Bootloader/SlimBootloader/sblplatform/merge_requests/164

Incase if PCR/hash config data is not available for a platform, it would use “PcdMeasuredBootHashType” to decide PCR.